easybuild/easyconfigs/l/LLVM/LLVM-5.0.0-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,46 @@
+# Contributed by Jean M. Favre (CSCS)
+# The build options and the version number were inherited from the VisIt build (as a guarantee that it works there)
+# http://visit.ilight.com/svn/visit/trunk/src/svn_bin/bv_support/bv_llvm.sh
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = "5.0.0"
+
+homepage = "http://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'dynamic': True}
```

Comment:
Dynamic is set True by default, so in theory having this line here is optional.

We could maybe consider to change EB and unset the `craype_link_type` if we choose static (to make sure that the user environment is not affecting the build).

Reply:
if set to True by default, then it should have worked the first time. I am still puzzled why it had failed.